docs/api.rst

```diff
@@ -4445,6 +4445,168 @@ Querying complete action history
    :statuscode 409: No user is currently logged in.
    :statuscode 500: Internal rotki error.
 
+
+Export PnL report debug data
+============================
+
+.. http:post:: /api/(version)/history/debug
```

Comment:
What do you think about an option to export to a json file? From my perspective that would be good since there possibly can be a ton of events and thus a very very huge response

Reply:
Because an async task can only return a dict, that's quite a limitation.

Reply:
Mm, why? What is wrong in writing to a file and then in a `dict` in the response returning a path to that file?

Reply:
Well, the disparity between the Docker & Electron version. One expects a blob(to be downloaded), the other provides a path for the data to be written to